test/e2e/handler/user_guide_test.go

```diff
@@ -44,13 +44,13 @@ var _ = Describe("Introduction", func() {
 			resetDesiredStateForNodes()
 		})
 
-		It("should succeed executing all the commands", func() {
+		It("[user-guide] should succeed executing all the commands", func() {
```

Comment:
Put [user-guide] at the `Describe` so we don't need to repeat it.

Reply:
Thanks, updated